core/router.py

```
@@ -0,0 +1,243 @@
+
+            # 解析 LLM 响应
+            import re
+            json_match = re.search(r'\{[^}]+\}', response, re.DOTALL)
+            if json_match:
+                result = json.loads(json_match.group())
+
+                return RoutingDecision(
+                    task_type=TaskType(result["task_type"]),
+                    confidence=result["confidence"],
+                    template_modules=self._get_template_modules(result["task_type"]),
+                    reasoning=f"LLM 路由决策：{result['reasoning']}"
```

Ruff (0.15.7)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)